CMakeLists.txt

```diff
@@ -27,16 +27,15 @@ include(ICUBcontribHelpers)
 include(ICUBcontribOptions)
 icubcontrib_set_default_prefix()
 
-
+## locate OpenCV libraries
 find_package(OpenCV REQUIRED)
-#set(OpenCV_INCLUDE_DIR "/usr/local/src/robot/opencv/include" "/usr/local/src/robot/opencv_contrib/modules/xfeatures2d/include") 
 include_directories(${OpenCV_INCLUDE_DIR})
 
 set(OpenCV_LIBRARIES "opencv_features2d;opencv_imgproc;opencv_core;opencv_cudastereo;opencv_ximgproc;opencv_xfeatures2d")
```

Comment:
`OpenCV_LIBRARIES` is set up automatically by the call to `find_package` and it should deal also with contrib modules.